modules/nf-core/cobs/compactconstruct/meta.yml

```diff
@@ -0,0 +1,60 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "cobs_compactconstruct"
+description: Builds a compact bloom filter COBS index
+keywords:
+  - COBS
+  - index
+  - k-mer index
+  - bloom filter
+tools:
+  - "cobs":
+      description: "Compact Bit-Sliced Signature Index (for Genomic k-Mer Data or q-Grams)"
+      homepage: "https://panthema.net/cobs"
+      documentation: "https://github.com/iqbal-lab-org/cobs"
+      tool_dev_url: "https://github.com/iqbal-lab-org/cobs"
+      doi: "10.1007/978-3-030-32686-9_21"
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
```

Comment:
please double check if this is actually a plausible example for the meta map.

Reply:
Thanks for bringing this to my attention, I'd always leave the default code created by `nf-core modules create` and wouldn't adapt it to the specific module. Fixed in https://github.com/nf-core/modules/pull/4607/commits/eedde7be21717dd51f0b6021831fb74bbca7ed51 and will also be fixed in my previous modules!